tests/Unit/NumericalAlgorithms/Interpolation/Test_InterpolationTargetReceiveVars.cpp

```diff
@@ -432,8 +471,10 @@ void test_interpolation_target_receive_vars() noexcept {
 
 SPECTRE_TEST_CASE("Unit.NumericalAlgorithms.InterpolationTarget.ReceiveVars",
                   "[Unit]") {
-  test_interpolation_target_receive_vars<MockPostInterpolationCallback, 1>();
+  test_interpolation_target_receive_vars<MockPostInterpolationCallback, 1, 0>();
   test_interpolation_target_receive_vars<MockPostInterpolationCallbackNoCleanup,
-                                         0>();
+                                         0, 0>();
+  test_interpolation_target_receive_vars<
+      MockPostInterpolationCallbackWithInvalidPoints<3>, 1, 3>();
 }
```

Comment:
Add an error test to confirm that it errors when the invalid points fill isn't specified.

Reply:
Adding an error test doesn't make sense in the current implementation.

If the invalid points fill isn't specified, the behavior is (currently) completely up to the callback.  

There are two (non-mock) callbacks right now: ObserveTimeSeriesOnSurface (which defines the fill to be quiet NaN) and FindApparentHorizon (which currently errors on unfilled points, but in a future PR it will adjust the position of the horizon and do another horizon iteration).